docs/user_manual.md

```diff
@@ -588,3 +571,42 @@ The name of a cluster is immutable, and it is included in the names of all of th
 4.  Delete the `sample-cluster` resource.
 
 At that point, you will be left with just the resources for `sample-cluster-2`. You can continue performing operations on `sample-cluster-2` as normal. You can also change or remove the `instanceIDPrefix` if you had to set it to a different value earlier in the process.
+
+# Coordinating Global Operations
+
+When running a FoundationDB cluster that is deployed across multiple Kubernetes clusters, each Kubernetes cluster will have its own instance of the operator working on the processes in its cluster. There will be some operations that cannot be scoped to a single Kubernetes cluster, such as changing the database configuration. The operator provides a locking system to ensure that only one instance of the operator can perform these operations at a time. You can enable this locking system by setting `lockOptions.disableLocks = false` in the cluster spec. The locking system is automatically enabled by default for any cluster that has multiple regions in its database configuration, or a `zoneCount` greater than 1 in its fault domain configuration.
+
+The locking system uses the `instanceIDPrefix` from the cluster spec to identify an instance of the operator. Make sure to set this to a unique value for each Kubernetes cluster, both to support the locking system and to prevent duplicate instance IDs.
+
+This locking system uses the FoundationDB cluster as its data source. This means that if the cluster is unavailable, no instance of the operator will be able to get a lock. If you hit a case where this becomes a blocker, you can disable the locking system by setting `lockOptions.disableLocks = true` in the cluster spec.
+
+In most cases, bounces will be done independently in each Kubernetes cluster, and the locking system will be used to ensure a minimum time between the different bounces and avoid multiple recoveries in a short span of time. During upgrades, however, all instances must be bounced at the same time. The operator will use the locking system to coordinate this. Each instance of the operator will store records indicating what processes it is managing and what version they will be running after the bounce. Each instance will then try to acquire a lock and confirm that every process reporting to the cluster is ready for the upgrade. If all processes are prepared, the operator will bounce all of them at once. If any instance of the operator is stuck and unable to prepare its processes for the upgrade, the bounce will not occur.
```

Comment:
I would prefer to use `restarted` in this case since `bounce` is an ambiguous term.